frontend/server/minio-helper.ts

```diff
@@ -102,7 +102,7 @@ function extractFirstTarRecordAsStream() {
   const extract = tar.extract();
   const transformStream = new Transform({
     write: (chunk: any, encoding: string, callback: (error?: Error | null) => void) => {
```

Comment:
nit: Replace unused parameter with `_`, as referenced in [this Stack Overflow post](https://stackoverflow.com/a/32198002) for example? (This is also a TypeScript suggestion on the VSCode UI.)
```suggestion
    write: (chunk: any, _, callback: (error?: Error | null) => void) => {
```